Cargo.toml

```diff
@@ -75,6 +74,8 @@ assert_cmd = "2.0"
 assert_fs = "1.0"
 axum = { version = "0.7.2", default-features = false, features = ["tokio", "http1"] }
 fs_extra = "1.3"
+gix-tempfile = "13.1.0"
```

Comment:
For some reason without this, the version used was `13.0.0` and test failed to build with a missing function definition.

Reply:
What are the steps to reproduce this issue? I commented this line out, run `cargo test`/`cargo test --all-features` and it seems to work.